bayesian/api_v1.py

```diff
@@ -736,16 +736,12 @@ def post():
             api_url = current_app.config['F8_API_BACKBONE_HOST']
             d = DependencyFinder()
             deps = {}
-            # TODO: don't use "true" and "false" for Boolean values
-            worker_flow_enabled = "false"
-            # TODO This will be changed once we add support for other ecosystems
+            worker_flow_enabled = False
 
-            if resolved_files_exist(manifests) == "true" \
-                    and ecosystem in ("npm", "pypi", "golang"):
-                # This condition for the flow from vscode
+            if resolved_files_exist(manifests) is True:
```

Comment:
this is same as `if resolved_files_exist(manifests):`

Reply:
done